metabase/metabase
# Improve text contrast in object detail in static embed

Closes https://github.com/metabase/metabase/issues/43532

### Description

Improve contrast in object detail card in static embed/public sharing. This should land in v51 since the foundation for theming isn't backported to v50.

### How to verify

1. Run `yarn storybook`
1. Open http://localhost:6006/?path=/story/embed-publicorembeddeddashboardview--card-visualizations-dark-theme

The contrast should look good.

### Demo

#### After
![image](https://github.com/user-attachments/assets/7b06b98e-2151-45f2-8801-eb3cded21800)

#### Before
![image](https://github.com/user-attachments/assets/1dbd38fb-6f2d-41a4-ae83-ce476898e389)

### Checklist
